libr/core/cmd_anal.c

```diff
@@ -3770,8 +3809,29 @@ static int cmd_anal_fcn(RCore *core, const char *input) {
 							pj_kn (pj, "to", ref->addr);
 							pj_end (pj);
 						} else {
-							r_cons_printf ("%c 0x%08" PFMT64x " -> 0x%08" PFMT64x "\n",
```

Comment:
why removing the arrow? I found it quite useful actually, but I don't know what others think...

Reply:
just a matter of taste. i can bring it back

Reply:
its back